item-search/README.md

```diff
@@ -112,6 +112,7 @@ parameter name is defined by the implementor and is not necessarily part of the
 
 ```json
 {
+    "stac_version": "1.0.0-beta.2",
```

Comment:
see https://github.com/radiantearth/stac-api-spec/issues/141

Reply:
@philvarner OK, #141 was opened after this PR. Version removed from example.